src/Psalm/Internal/Analyzer/Statements/Block/IfElse/ElseIfAnalyzer.php

```diff
@@ -116,25 +115,19 @@ function (Clause $c) use ($mixed_var_ids, $elseif_cond_id): Clause {
             $elseif_clauses
         );
 
-        $entry_clauses = array_map(
-            /**
-             * @return Clause
-             */
-            function (Clause $c) use ($assigned_in_conditional_var_ids, $elseif_cond_id): Clause {
-                $keys = array_keys($c->possibilities);
+        $entry_clauses = [];
 
-                foreach ($keys as $key) {
-                    foreach ($assigned_in_conditional_var_ids as $conditional_assigned_var_id => $_) {
-                        if (preg_match('/^' . preg_quote($conditional_assigned_var_id, '/') . '(\[|-|$)/', $key)) {
-                            return new Clause([], $elseif_cond_id, $elseif_cond_id, true);
-                        }
+        foreach ($if_conditional_scope->entry_clauses as $c) {
+            foreach ($c->possibilities as $key => $_value) {
+                foreach ($assigned_in_conditional_var_ids as $conditional_assigned_var_id => $_) {
+                    if (preg_match('/^'.preg_quote($conditional_assigned_var_id, '/').'(\[|-|$)/', $key)) {
+                        $entry_clauses[] =  new Clause([], $elseif_cond_id, $elseif_cond_id, true);
```

Comment:
One more :(

Reply:
`break 2` is already there.

Reply:
Sorry, what do you mean?

Reply:
I added it for ifelse, but this is elseif analyzer

Reply:
Yeah, my bad, ignore my comment.